content/shared/influxdb3-cli/create/table.md

```diff
@@ -34,6 +35,22 @@ influxdb3 create table [OPTIONS] \
 |        | `--tls-ca`   | Path to a custom TLS certificate authority (for testing or self-signed certificates)     |
 | `-h`   | `--help`     | Print help information                                                                   |
 |        | `--help-all` | Print detailed help information                                                          |
+{{% /hide-in %}}
+
+<!-- Using the show-in shortcode for only the retention-period option breaks the formatting in Core -->
+{{% show-in "enterprise" %}}
+| Option |                      | Description                                                                                                                                      |
+| :----- | :------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-H`   | `--host`             | Host URL of the running {{< product-name >}} server (default is `http://127.0.0.1:8181`)                                                         |
+| `-d`   | `--database`         | _({{< req >}})_ Name of the database to operate on                                                                                               |
+|        | `--retention-period` | [Retention period](/influxdb3/version/reference/glossary/#retention-period) ([duration](/influxdb3/version/reference/glossary/#duration) value, for example: `30d`, `24h`, `1h`) for data in the table|
```

Comment:
In previous products, the default is `0h` (infinite). I don't know if that's true here, but we probably should call out that `0h` means infinite. If not here, then in the Admin docs we update/create for retention periods.

Reply:
> In previous products, the default is `0h` (infinite). I don't know if that's true here, but we probably should call out that `0h` means infinite. If not here, then in the Admin docs we update/create for retention periods.

See #6167